docker/.env

```diff
@@ -6,8 +6,20 @@ MF_NGINX_SSL_PORT=443
 MF_NGINX_MQTT_PORT=1883
 MF_NGINX_MQTTS_PORT=8883
 
-## NATS
-MF_NATS_URL=nats://nats:4222
+# Message Broker
+MF_BROKER_TYPE=nats
+MF_BROKER_URL=nats://broker:4222
```

Comment:
The broker URL needs to change depending on the broker type - here, it's hardcoded to use NATS.